azure-mgmt-scheduler/src/main/java/com/microsoft/azure/management/scheduler/Job.java

```diff
@@ -93,33 +98,217 @@
          */
         interface WithStartTime {
             /**
+             * A job which will start right away.
+             *
+             * @return the next stage of the definition
+             */
+            WithRecurrenceOrAction startingNow();
+
+            /**
              * Specifies the start time of the job.
              *
              * @param startTime the start time
              * @return the next stage of the definition
              */
-            WithRecurrenceOrAction withStartTime(DateTime startTime);
+            WithRecurrenceOrAction startingAt(DateTime startTime);
         }
 
         /**
          * The stage of the job definition for an Azure Scheduler allowing to specify the recurrence and action details.
          */
-        interface WithRecurrenceOrAction {
+        interface WithRecurrenceOrAction extends WithAction {
             /**
-             * Specifies the recurrence  of the job.
+             * The recurrence object specifies recurrence rules for the job and the recurrence the job will execute with.
+             * <p>
+             * The recurrence object supports the elements frequency, interval, endTime, count, and schedule. If recurrence
```

Comment:
we don't want to use code-like sounding wording here. Better use just plain language:
"The recurrent object allows the frequency, the end time, the number of occurrences and the schedule to be specified. If a recurrence is defined, at least the frequency should be specified, the rest is optional."